### PR TITLE
Enable PDO Drivers for Auth Module

### PR DESCRIPTION
**Pull request by BlackScorp**
After trying Kohanas HMVC i wanted to use the Official Auth Module. But since i was using PDO , Kohana throw following Exception

throw new Kohana_Exception('Database method :method is not supported by :class',
            array(':method' => **FUNCTION**, ':class' => **CLASS**));

So i tryed to fix it. Acually Register/Login/Logout works now in my Test installation. And sorry to made a pull Request to master

### CODE REVIEW

File: classes/kohana/database/pdo.php [hunk 1]
```diff
@@ -232,14 +232,102 @@ class Kohana_Database_PDO extends Database {
...
-		throw new Kohana_Exception('Database method :method is not supported by :class',
-			array(':method' => __FUNCTION__, ':class' => __CLASS__));
+		  // Make sure the database is connected
+        $this->_connection or $this->connect();
+        // Quote the table name
+        $table = ($add_prefix === TRUE) ? $this->quote_table($table) : $table;
+
+        if (is_string($like)) {
+            // Search for column names
+            $result = $this->_connection->query('SHOW FULL COLUMNS FROM ' . $table . ' LIKE ' . $this->quote($like));
```
Comment (isaiahdw) [inline classes/kohana/database/pdo.php:262]: This doesn't work with all databases, so I don't think including it is a good idea. It's up to you to implement a solution for the database you are using. PDO doesn't provide a way to list the columns, so there isn't anyway for us to either.

Comment (BlackScorp) [inline classes/kohana/database/pdo.php:262]: ah yeah youre right. the alternative way would to get the columns from information_schema table, which is installed on mysql,mssql,mysqlight by default oracle has his own table where table/column names are stored... but i think a kohana schould say that is still possible to use PDO and Auth module by adding protected $_table_columns in the model. else everyone will think like me that Kohanas Auth does not support PDO

Comment (isaiahdw) [inline classes/kohana/database/pdo.php:262]: That sounds like a good idea. If you could update the docs to add that note and create a pull request that would be great!

Comment (BlackScorp) [inline classes/kohana/database/pdo.php:262]: i can add a new pull request with another Exception Error but i dont know if i have the rights to update docs. or do you mean to make a pull request with infos from information_schema?

**Pull request closed without merging**